packages/@vuepress/plugin-active-header-links/src/client/composables/useActiveHeaderLinks.ts

```diff
@@ -107,13 +107,14 @@ export const useActiveHeaderLinks = ({
   }
 
   const onScroll = debounce(() => setActiveRouteHash(), delay)
+  const listener = (): Promise<void> => onScroll()
```

Comment:
What about normalizing those var names?

onScroll -> setActiveRouteHashDebounce
listener -> onScroll

Or any better advices ?

Reply:
This might look better? 🤔 

```ts
const setActiveRouteHash = debounce(() => { ... }, delay)
const onScroll = (): Promise<void> => setActiveRouteHash()
```

Reply:
Yes